internal/pkg/cli/app.go

```diff
@@ -4,11 +4,15 @@
 package cli
 
 import (
+	"errors"
+
 	"github.com/aws/amazon-ecs-cli-v2/cmd/archer/template"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli/group"
 	"github.com/spf13/cobra"
 )
 
+var errNoWorkspace = errors.New("could not find a project attached to this workspace, please run `project init` first")
```

Comment:
Should this be called `errNoProject` instead? 🤔

Reply:
That makes sense, changed to `errNoProjectInWorkspace`!